keith-vscode/src/error-handler.ts

```diff
@@ -27,6 +27,7 @@ export class KeithErrorHandler implements ErrorHandler {
 
     error(error: Error, message: Message, count: number): ErrorAction {
         window.showErrorMessage('Connection to KIELER Language Server produced an error!')
+        // eslint-disable-next-line no-console
         console.error(error)
```

Comment:
I guess showErrorMessage should be enough and console.error is not needed here, right?

Reply:
I'd say printing the error to the console as well is fine here, if the UI did not respond anymore or the error messaging does not work. This way it could still be retrieved from the console. But I would not mind to remove this as well. I've externalized this into #62 earlier to have a better look at the statements here, this PR is mainly here to fix the linting process, not question some of the choices made here immedeately.